src/component_managers/astt_comp_manager.py

```diff
@@ -3,7 +3,8 @@
 import time
 
 import canopen
-from sources import Sun
+
+from .sources import Sun
```

Comment:
I think we should specify for a person who wants to run this on their local machine to check that the PYTHONPATH points to "home/usr/../../src/"

Reply:
In the Pull request or as a code comment?